jme3-core/src/main/java/com/jme3/math/Plane.java

```diff
@@ -291,4 +291,13 @@ public Plane clone() {
             throw new AssertionError();
         }
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;

```

Comment:
Also please use curly Braces for if statement. 
```
if (this == o) {
     return true;
}
```

It is recommended to use the default code formatter of your IDE if you are in Netbeans or Eclipse.